### PR TITLE
Validate collection

### PR DESCRIPTION
**Pull request by elsiosanchez**
- [x] add charge button
- [x] add collection validation 
- [x] correcting the currency converter
- [X] add translation

collage

### CODE REVIEW

File: src/components/ADempiere/Form/VPOS/Collection/index.vue [hunk 1]
```diff
@@ -472,31 +524,31 @@ export default {
       let label
       switch (type) {
         case 'A':
-          label = 'Depósito directo'
+          label = this.$t('form.pos.collect.directDeposit')
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Collection/index.vue]: Please chage it for value

**Comment by yamelsenih** [comment]
Please resolve conflicts

File: src/utils/ADempiere/apiConverts/core.js [hunk 2]
```diff
@@ -154,6 +155,23 @@ export function convertBusinessPartner(businessPartnerToConvert) {
...
 export function convertConversionRate(conversionRateToConvert) {
   const { id, uuid } = conversionRateToConvert
+  if (isEmptyValue(conversionRateToConvert.currency_from) && isEmptyValue(conversionRateToConvert.currency_from)) {
+    return {
+      uuid,
+      id,
+      conversionTypeUuid: conversionRateToConvert.conversion_type_uuid,
+      validFrom: conversionRateToConvert.valid_from,
+      validTo: conversionRateToConvert.valid_to,
+      // currencyFrom: convertCurrency(
```
Comment (yamelsenih) [inline src/utils/ADempiere/apiConverts/core.js]: Remove this

File: src/store/modules/ADempiere/pointOfSales/collection.js [hunk 3]
```diff
@@ -25,11 +25,12 @@ const collection = {
      * creating boxes with the payment list
      */
     setPaymentBox({ state, commit, getters }, params) {
-      const payments = getters.getPaymentBox.find(element => {
-        if (element.tenderType === params.tenderType && element.currency.id === params.currency.id) {
-          return element
-        }
-      })
+      // const payments = getters.getPaymentBox.find(element => {
+      //   if (element.tenderType === params.tenderType && element.currency.id === params.currency.id) {
```
Comment (yamelsenih) [inline src/store/modules/ADempiere/pointOfSales/collection.js]: Remove this

File: src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js [hunk 4]
```diff
@@ -50,9 +50,15 @@ export default [
     elementColumnName: 'DateTrx',
     isFromDictionary: true,
     overwriteDefinition: {
+      // handleActionKeyPerformed: true,
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js]: Remove this

File: src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js [hunk 5]
```diff
@@ -50,9 +50,15 @@ export default [
...
+      // handleActionPerformed: true,
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js]: Remove this

**Review by yamelsenih — CHANGES_REQUESTED** [review]
Remove this

File: src/store/modules/ADempiere/pointOfSales/collection.js (hunk 3 above)
Comment (elsiosanchez) [reply src/store/modules/ADempiere/pointOfSales/collection.js]: code snippet removed

File: src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js (hunk 4 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js]: code snippet removed

File: src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js (hunk 5 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS/Collection/fieldsListCollection.js]: code snippet removed

**Pull request merged**